business_objects/record_label_association.py

```diff
@@ -199,6 +199,28 @@ def get_all_classifications_for_information_source(
     )
 
 
+def get_existing_record_label_associations(
+    project_id: str, record_ids: List[str], user_ids: List[str], label_ids: List[str]
+) -> List[RecordLabelAssociation]:
+    return (
+        session.query(RecordLabelAssociation)
+        .filter(RecordLabelAssociation.project_id == project_id)
+        .filter(RecordLabelAssociation.record_id.in_(record_ids))
+        .filter(
+            RecordLabelAssociation.created_by.in_(
+                [
+                    user_id
+                    for user_id in user_ids
+                    if user_id != enums.RecordImportMappingValues.IGNORE.value
```

Comment:
this feels like a very specific exclude for an unspecific case.
if we give a list of user ids then I would expect that the list isn't further processed. Maybe remove IGNORE beforehand

Reply:
changed so the filtering of IGNORE is done before calling of method